text-generation-inference/docker/Dockerfile

```diff
@@ -25,6 +26,8 @@ COPY --from=tgi /tgi/launcher launcher
 RUN cargo chef prepare --recipe-path recipe.json
 
 FROM chef AS builder
+ARG ENABLE_GCP_INTEGRATION
```

Comment:
I would rename this, as TPUs will _always_ run on GCP. Maybe "GOOGLE_FEATURE" to be in line with the naming in TGI?

Reply:
OK